src/rpc/rawtransaction.cpp

```diff
@@ -976,6 +1025,52 @@ UniValue createrawcertificate(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Missing mandatory parameter in input: \"scProof\"" );
     }
 
+    // ---------------------------------------------------------
+    // just check against a maximum size TODO for the time being set to 32 K
+    static const size_t MAX_FE_SIZE_BYTES  = 1024*32;
```

Comment:
For the compressed FE max size equals to the ScFieldElement size. Currently it is 96 bytes (actually 5 bits less).
With the new curve it will be 32 bytes (actually 253 bits)

Reply:
Ok, the check is now against a maximum SC_FIELD_ELEMENT size